core/capabilities/ccip/oraclecreator/plugin.go

```diff
@@ -307,6 +304,12 @@ func (i *pluginOracleCreator) createFactoryAndTransmitter(
 		return nil, nil, fmt.Errorf("unsupported chain %v", chainFamily)
 	}
 	messageHasher := plugin.MessageHasher(i.lggr.Named(chainFamily).Named("MessageHasherV1"))
+	addressCodec := ccipcommon.NewAddressCodec(
+		ccipcommon.NewAddressCodecParams(
```

Comment:
Why do you need to send these?
Why can't the NewAddressCodecParams() just create locally itself?